bindings/pyroot/pythonizations/src/RDataFramePyz.cxx

```diff
@@ -22,6 +22,7 @@
 
 ////////////////////////////////////////////////////////////////////////////
 /// \brief Make an RDataFrame from a dictionary of numpy arrays
+/// \param[in] self null
```

Comment:
Thanks for catching this, can you keep the `Always null, since this is a module function.` that is written for other module functions instead of just `null`.

Reply:
Changed :)